cronjob.py

```diff
@@ -0,0 +1,81 @@
+
+from lib.modules.networkingmodule.networkingresponse import *
+import requests
+import json
+import numpy as np
+import os
+from dotenv import load_dotenv
+from pathlib import Path
+from aiohttp import ClientError
+from lib.modules.databasemodule.database import database
+db = database.instance()
+
+env_path = Path('.') / '.env'
+load_dotenv(dotenv_path=env_path)
+
+if __name__ == '__main__':
+    try:
+        header = {'Authorization': 'Bearer ' + os.getenv("SLACK_BOT_TOKEN")}
+        # Gets list of users in Networking channel
+        data = {'channel': 'C03QSC88Y4E'}
+        rawusers = requests.get('https://slack.com/api/conversations.members', headers=header, params=data).content
+        if not json.loads(rawusers.decode('utf-8'))['ok']:
+            print("Error: " + json.loads(rawusers.decode('utf-8'))['error'])
+        users = json.loads(rawusers.decode('utf-8'))['members']
+        users.remove('U03QC4N6TDF')
+        # Determines if users are alum
+        alum = []
+        student = []
+        for user in filter(lambda x: x != 'U03THUJ5Y2X', users):
+            if db.networking_is_alum(user):
+                alum.append(user)
+            else:
+                student.append(user)
+        if (len(alum) + len(student)) % 2 != 0:
+            if len(alum) > 0:
+                popped_user = alum.pop()
+            else:
+                popped_user = student.pop()
+            popped_response = requests.post('https://slack.com/api/conversations.open', headers=header, data={"users": popped_user}).content
+            popped_data = {'channel': json.loads(popped_response.decode('utf-8'))['channel']['id'], 'text': 'You are the odd one out... :cry:'}
```

Comment:
Let's change this message to something like "Matching for networking this week has been done but unfortunately we had an odd number of users and you were unable to be paired. We will try to get you into a call next week!" Additionally, potentially might be smart to shuffle the stacks bc I'm concerned that the same person could be the odd one out several weeks in a row